app/src/main/java/woowacourse/movie/activity/SeatPickerActivity.kt

```diff
@@ -0,0 +1,224 @@
+package woowacourse.movie.activity
+
+import android.app.AlertDialog
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import android.widget.TableLayout
+import android.widget.TableRow
+import android.widget.TextView
+import android.widget.Toast
+import androidx.core.view.children
+import com.woowacourse.domain.seat.Seat
+import com.woowacourse.domain.seat.SeatColumn
+import com.woowacourse.domain.seat.SeatGroup
+import com.woowacourse.domain.seat.SeatRow
+import com.woowacourse.domain.ticket.Ticket
+import com.woowacourse.domain.ticket.TicketBundle
+import woowacourse.movie.BundleKeys
+import woowacourse.movie.R
+import woowacourse.movie.getSerializableCompat
+import woowacourse.movie.mapper.toDomain
+import woowacourse.movie.mapper.toPresentation
+import woowacourse.movie.model.SeatGroupModel
+import woowacourse.movie.movie.MovieBookingInfo
+import woowacourse.movie.movie.MovieBookingSeatInfo
+
+class SeatPickerActivity : BackButtonActivity() {
+    private val seatTableLayout: TableLayout by lazy { findViewById(R.id.tl_seats) }
+    private val pickDoneButton: Button by lazy { findViewById(R.id.bt_seat_picker_done) }
+    private val seatPickerTicketPrice: TextView by lazy { findViewById(R.id.tv_seat_picker_ticket_price) }
+    private val movieTitle: TextView by lazy { findViewById(R.id.tv_seat_picker_movie) }
+    private var seatGroup = SeatGroup()
+    private lateinit var ticketBundle: TicketBundle
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_seat_picker)
+
+        val movieBookingInfo = getMovieBookingInfo()
+
+        initSeatName()
+        initSeatNameColor()
+        initMovieTitle(movieBookingInfo)
+        ticketBundle = TicketBundle(count = movieBookingInfo.ticketCount)
+        reloadData(savedInstanceState)
+
+        setPickDoneButtonClickListener()
+        setSeatsOnClickListener(movieBookingInfo)
+    }
+
+    private fun getMovieBookingInfo(): MovieBookingInfo {
+        return intent.getSerializableCompat(BundleKeys.MOVIE_BOOKING_INFO_KEY)
+            ?: MovieBookingInfo.dummyData
+    }
+
+    private fun initSeatName() {
+        getRowSeats().forEachIndexed { index, rowSeat ->
+            rowSeat.children
+                .filterIsInstance<TextView>()
+                .forEachIndexed { rowIndex, seat ->
+                    seat.text = formatSeatName(index, rowIndex)
+                }
+        }
+    }
+
+    private fun formatSeatName(index: Int, rowIndex: Int) =
+        getString(R.string.seat_name).format(ALPHABET[index], rowIndex + 1)
```

Comment:
```suggestion
        getString(
            R.string.seat_name,
            ALPHABET[index], 
            rowIndex + 1
        )
```
String.format 함수를 사용하셔도 좋지만, getString 함수 자체에서도 format을 지원하고 있어요.